.circleci/config.yml

```diff
@@ -17,11 +17,15 @@ jobs:
       - run:
           name: Push images
           command: |
-            docker push quay.io/$CIRCLE_PROJECT_USERNAME/cluster-api-azure-controller:$CIRCLE_SHA1
+            docker push quay.io/$CIRCLE_PROJECT_USERNAME/cluster-api-azure-controller:${CIRCLE_TAG:-$CIRCLE_SHA1}
 workflows:
   version: 2
   build_and_update:
     jobs:
       - build:
           context:
             - architect
+          # Trigger the job also on git tag.
+          filters:
+            tags:
+              only: /^v[0-9]+\.[0-9]+\.[0-9]+(?:-[0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*)?(?:\+[0-9A-Za-z-]+)?$/
```

Comment:
docker build fails if the docker tag is not a semver.
This is a regexp to validate semver strings

Reply:
uh nice! Can you maybe document this in the architect-orb repository somewhere? Somewhere here? https://github.com/giantswarm/architect-orb/tree/master/docs